ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksTable.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { generatePath, Link } from 'react-router-dom';
+import {
+    Divider,
+    Progress,
+    ProgressMeasureLocation,
+    Text,
+    TextVariants,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+    Tooltip,
+} from '@patternfly/react-core';
+import { Table, TableText, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import { ComplianceCheckResultStatusCount } from 'services/ComplianceResultsService';
+import { getTableUIState } from 'utils/getTableUIState';
+import TbodyUnified from 'Components/TableStateTemplates/TbodyUnified';
+
+import { coverageCheckDetailsPath } from './compliance.coverage.routes';
+import CoverageTableViewToggleGroup from './components/CoverageTableViewToggle';
+import StatusCountIcon from './components/StatusCountIcon';
+import {
+    calculateCompliancePercentage,
+    getCompliancePfClassName,
+    getStatusCounts,
+} from './compliance.coverage.utils';
```

Comment:
nit: Usually, I try to organize the imports like so:

1. Third party libraries
2. Utils, Services, Hooks, etc.
3. Components

Makes it easier to parse

Reply:
Oh I've been doing:

1. Third party library
2. Absolute imports
3. Relative imports

but I don't have a strong preference

Reply:
Ditto what Brad said. Most of the code follows that pattern as far as I an remember.

Reply:
I do similar to what Brad said as well. I have noticed lately that I am spending quite a bit of time manually reordering imports, which to me should be a Prettier type task that we don't think about at all.

Especially with Node and Platform CVEs, which uses code under `Containers/Vulnerabilities` that I would expect to auto-import with relative syntax, but VSCode keeps using absolute syntax.

I'd like to thing there is a way we can set up better autoformatting of imports and then just forget about it. We would probably need it to:
- Not delete the top level `React` import
- Not fail builds when things are incorrect
- Not format files other than the ones we are currently working on, to avoid conflicts

Reply:
Ah looks like we're all doing it a bit differently. Maybe worth a discussion on Friday to make sure everyone's on the same page

Reply:
I agree that it is a bit of a hassle working with the relative and absolute imports together. I think ESLint automatically sets relative imports below absolute ones. However, I find that there is a lack of contextual grouping. Imports are then sorted by their paths rather than by their purpose or functionality. Which I feel makes the file's dependencies harder to understand at a glance. For larger files with many imports, it would be even harder to understand. Usually, I will just convert the relative to absolute paths and then order them. I know everyone seems to be going one way, but wanted to share my reasoning and see what people thought.